src/test_workflow/integ_test/service_opensearch.py

```diff
@@ -50,11 +56,31 @@ def __download(self):
         bundle_name = self.dependency_installer.download_dist(self.work_dir)
         logging.info(f"Downloaded bundle to {os.path.realpath(bundle_name)}")
 
-        logging.info(f"Unpacking {bundle_name}")
-        with tarfile.open(bundle_name, 'r') as bundle_tar:
-            bundle_tar.extractall(self.work_dir)
-
-        logging.info(f"Unpacked {bundle_name}")
+        logging.info(f"Installing {bundle_name} in {self.install_dir}")
+
+        if self.distribution == "tar":
```

Comment:
Subclass `ServiceOpenSearch` into `ServiceOpenSearchTar` and ``ServiceOpenSearchRpm` and implement the differences there.